src/azul/terra.py

```diff
@@ -272,8 +283,23 @@ def run_sql(self, query: str) -> BigQueryRows:
                     sleep(delay)
                 else:
                     raise e
+            else:
+                if log.isEnabledFor(logging.DEBUG):
+                    log.debug('Job info: %r', self._job_info(result, job))
```

Comment:
You shouldn't use repr here, but json.dumps. Using repr interferes with CW's JSON parser. Just drop the indent=4 to avoid adding more newlines.